core/domain/stats_jobs_one_off.py

```diff
@@ -1046,6 +1046,26 @@ def reduce(key, stringified_values):
         yield '%s: %d' % (key, len(stringified_values))
 
 
+class RefreshInteractionRegistryJob(jobs.BaseMapReduceJobManager):
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [exp_models.ExplorationModel]
+
+    # pylint: disable=unused-argument
+    @staticmethod
+    def map(item):
+        yield ('key', 'value')
+
+    @staticmethod
+    def reduce(key, stringified_values):
+        # pylint: disable=protected-access
+        interaction_registry.Registry._refresh()
```

Comment:
It seems incredibly heavyweight to do this using an MR job. Why not just click a button in the admin dashboard that performs this action? I don't understand the point of iterating through all the explorations...